src/execution_plan/execution_plan.h

```diff
@@ -107,6 +110,9 @@ void ExecutionPlan_PlaceFilterOps(ExecutionPlan *plan, const OpBase *recurse_lim
 /* Retrieve the map of aliases to Record offsets in this ExecutionPlan segment. */
 rax *ExecutionPlan_GetMappings(const ExecutionPlan *plan);
 
+/* Retrieve the result set column names. */
+const char **ExecutionPlan_GetResultColumns(const ExecutionPlan *plan);
```

Comment:
Even if you wish to keep this function, it doesn't need to be exposed in the header.

Reply:
removed